Neos.Flow/Scripts/Migrations/Git.php

```diff
@@ -94,9 +94,9 @@ public static function remove($fileOrDirectory)
     {
         $result = 255;
         if (is_dir($fileOrDirectory)) {
-            system('git rm -qr ' . escapeshellarg($fileOrDirectory), $result);
+            exec('git rm -qr ' . escapeshellarg($fileOrDirectory), $output, $result);
```

Comment:
Well, we want the status code, but as that is the third argument I also need to give the second (See reference about exec). I guess that's why system was used in the first place. But unifying the usage is worth even if we assign a useless variable.